src/compute-pr-actions.ts

```diff
@@ -88,13 +94,12 @@ const uriForTestingNewPackages = `${baseURI}#testing`;
 const uriForDefinitionOwners = `${baseURI}#definition-owners`;
 const uriForWorkflow = `${baseURI}#make-a-pull-request`;
 
-const enum Staleness {
-    Fresh,
-    PayAttention,
-    NearlyYSYL,
-    YSYL,
-    NearlyAbandoned,
-    Abandoned,
+type Staleness = {
+    readonly kind: StalenessKind;
+    readonly days: number;
+    readonly state: "fresh" | "attention" | "nearly" | "done";
+    readonly explanation?: string;
+    readonly doTimelineActions: (context: Actions) => unknown;

```

Comment:
Just curious: I'd've expected `void`?
```suggestion
    readonly doTimelineActions: (context: Actions) => void;

Reply:
Yeah, `void` is better.